README.md

```diff
@@ -1,23 +1,64 @@
 # Flowshutter
 
-Flowshutter is a customize camera remote written in micropython.
+<p align="center">
+  <a href="https://gyroflow.xyz">Homepage</a> •
+  <a href="https://discord.gg/WfxZZXjpke">Discord</a> •
+  <a href="https://github.com/gyroflow/flowshutter/issues">Report bug</a> •
+  <a href="https://github.com/gyroflow/flowshutter/issues">Request feature</a>
+</p>
 
-It can synchronize vedio recording of the camera and blackbox logging of your motion logger (in most cases betaflight/emuflight controller with flash chip or SD card). 
+<p align="center">
+  <a href="https://github.com/gyroflow/flowshutter/languages/top">
+    <img src="https://img.shields.io/github/languages/top/gyroflow/flowshutter" alt="Languages">
+  </a>
+  <a href="https://github.com/gyroflow/flowshutter/">
+    <img src="https://img.shields.io/github/contributors/gyroflow/flowshutter?color=dark-green" alt="Contributors">
+  </a>
+  <a href="https://github.com/gyroflow/flowshutter/issues/">
+    <img src="https://img.shields.io/github/issues/gyroflow/flowshutter" alt="Issues">
+  </a>
+  <a href="https://github.com/gyroflow/flowshutter/blob/master/LICENSE">
+    <img src="https://img.shields.io/github/license/gyroflow/flowshutter" alt="License">
+  </a>
+</p>
 
-It was designed to be used with gyroflow software to provide you the best open source video stabilization experience.
+Flowshutter is a customize camera remote, designed to be the most flexible and reliable gyroflow camera external motion logger. It can provide precise synchronization of camera vedio recording and motion logger (betaflight/emuflight FC) recording. 
 
-## Hardware
+It was designed to be used with gyroflow software to provide you one of the best open source video stabilization experience.
 
-Community version can be found on [here](https://oshwhub.com/AirFleet/xiang-ji-kong-zhi-ban). Checkout our [Build Video](https://www.youtube.com/watch?v=ELaQPYE9ncA)!
 
-## Supported camera
+## Features
 
-- Sony camera: only with Multiport USB protocol (works beautifully on my A6300)
-- Others is on the way
+Flowshutter has the following features:
 
-## Supported FC
+- '1-click' - (1) start/stop camera recording and (2) arm/disarm FC, via one click 
+- Camera recording start/stop control
+- FC arm/disarm control
+- OLED display
+
+with many more features on the way!
+
+## Supported Hardware
+
+Flowshutter haven't worked any manufactures yet. So there's no "Ready-to-Use" commercial hardware\product yet.
```

Comment:
Flowshutter haven't worked with any manufacturers yet. So there's no "Ready-to-Use" commercial hardware/product yet.

Reply:
accepted